test/integration/sequelize.test.js

```diff
@@ -1402,7 +1480,12 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
             }).then(() => {
               return this.User.findAll({ transaction: this.transaction });
             }).then(users => {
-              expect(users).to.have.length(0);
+              if (dialect === 'db2') {
+                // SAVE commits in db2. There is no odbc API for save command.
+                expect(users).to.have.length(1);
```

Comment:
Updated comment in the code. Db2 value is different than other db because, db2 do not have support for nested transaction and save transaction commits in db2.